generated/proto/metricpb/composite.proto

```diff
@@ -43,8 +43,8 @@ message GaugeWithMetadatas {
   StagedMetadatas metadatas = 2 [(gogoproto.nullable) = false];
 }
 
-message TimedMetricWithForwardMetadata {
-  TimedMetric metric = 1 [(gogoproto.nullable) = false];
+message ForwardedMetricWithMetadata {
```

Comment:
So we will be having a separate message for timed/carbon metric?

Reply:
Yes, that's the plan.